wallet/wallet.go

```diff
@@ -498,6 +498,11 @@ out:
 				continue
 			}
 			timeout = req.lockAfter
+			if timeout == nil {
+				log.Info("Wallet unlocked without timeout")
+			} else {
+				log.Info("Wallet temporarily unlocked")
```

Comment:
We were both editing at the same time.  Perhaps the duration should be passed along in the `unlockRequets` so the log has the necessary information?

I understand it's advisory, but it's still useful.  There are really only two cases where it will be wrong:

1) It happens to be on hold the moment it expires, but even then, it will be unlocked as soon as the hold has been released, so it won't be that much of a difference
2) The wallet is manually locked before the time.  That is also fine, in my opinion, because the verbiage says "set to expire in %v".  Just because it is manually locked before it expired doesn't change the fact it is being unlocked with a timeout of X.

Anyways, it's not a big deal, though I would certainly prefer it to be more explicit in the logging.  I see a message like "The wallet has been unlocked temporarily" and the first thing that comes to my mind is "FOR HOW LONG????!??!?!"